### PR TITLE
[Angular] Minor style code

### PR DESCRIPTION
**Pull request by qmonmert**
Improve some code style

---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [ ] Tests are added where necessary
- [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) was updated if necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: generators/client/templates/angular/src/main/webapp/app/app-routing.module.ts.ejs [hunk 1]
```diff
@@ -18,6 +18,7 @@
 -%>
 import { NgModule } from '@angular/core';
 import { RouterModule } from '@angular/router';
+
```
Comment (kaidohallik) [inline generators/client/templates/angular/src/main/webapp/app/app-routing.module.ts.ejs:21]: Those empty lines are intentional, those are separating library and application internal imports.

File: generators/client/templates/angular/src/main/webapp/app/config/uib-pagination.config.ts.ejs [hunk 2]
```diff
@@ -18,6 +18,7 @@
 -%>
 import { Injectable } from '@angular/core';
 import { NgbPaginationConfig } from '@ng-bootstrap/ng-bootstrap';
+
```
Comment (kaidohallik) [inline generators/client/templates/angular/src/main/webapp/app/config/uib-pagination.config.ts.ejs:21]: Intentional blank line.

File: generators/client/templates/angular/src/main/webapp/app/core/util/alert.service.ts.ejs [hunk 3]
```diff
@@ -20,6 +20,7 @@ import { Injectable, SecurityContext, NgZone } from '@angular/core';
 import { DomSanitizer } from '@angular/platform-browser';
 <%_ if (enableTranslation) { _%>
 import { TranslateService } from '@ngx-translate/core';
+
```
Comment (kaidohallik) [inline generators/client/templates/angular/src/main/webapp/app/core/util/alert.service.ts.ejs:23]: Intentional blank line.

File: generators/client/templates/angular/src/main/webapp/app/layouts/profiles/page-ribbon.component.ts.ejs [hunk 4]
```diff
@@ -19,6 +19,7 @@
 import { Component, OnInit } from '@angular/core';
 import { Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
+
```
Comment (kaidohallik) [inline generators/client/templates/angular/src/main/webapp/app/layouts/profiles/page-ribbon.component.ts.ejs:22]: Intentional blank line.

File: generators/client/templates/angular/src/main/webapp/app/login/login.component.ts.ejs [hunk 5]
```diff
@@ -19,6 +19,7 @@
 import { Component, ViewChild, OnInit, AfterViewInit, ElementRef } from '@angular/core';
 import { FormBuilder, Validators } from '@angular/forms';
 import { Router } from '@angular/router';
+
```
Comment (kaidohallik) [inline generators/client/templates/angular/src/main/webapp/app/login/login.component.ts.ejs:22]: Intentional blank line.

File: generators/client/templates/angular/src/main/webapp/app/login/login.route.ts.ejs [hunk 6]
```diff
@@ -17,6 +17,7 @@
  limitations under the License.
 -%>
 import { Route } from '@angular/router';
+
```
Comment (kaidohallik) [inline generators/client/templates/angular/src/main/webapp/app/login/login.route.ts.ejs:20]: Intentional blank line.

File: generators/client/templates/angular/src/main/webapp/app/shared/shared.module.ts.ejs [hunk 7]
```diff
@@ -17,6 +17,7 @@
  limitations under the License.
 -%>
 import { NgModule } from '@angular/core';
+
```
Comment (kaidohallik) [inline generators/client/templates/angular/src/main/webapp/app/shared/shared.module.ts.ejs:20]: Intentional blank line.

**Review by kaidohallik — COMMENTED** [review]
Blank lines between library and internal module imports are intentional.
Other changes are OK.

File: generators/client/templates/angular/src/main/webapp/app/app-routing.module.ts.ejs (hunk 1 above)
Comment (qmonmert) [reply generators/client/templates/angular/src/main/webapp/app/app-routing.module.ts.ejs:21]: yes intentional

File: generators/client/templates/angular/src/main/webapp/app/config/uib-pagination.config.ts.ejs (hunk 2 above)
Comment (qmonmert) [reply generators/client/templates/angular/src/main/webapp/app/config/uib-pagination.config.ts.ejs:21]: yes intentional

File: generators/client/templates/angular/src/main/webapp/app/core/util/alert.service.ts.ejs (hunk 3 above)
Comment (qmonmert) [reply generators/client/templates/angular/src/main/webapp/app/core/util/alert.service.ts.ejs:23]: yes intentional

File: generators/client/templates/angular/src/main/webapp/app/layouts/profiles/page-ribbon.component.ts.ejs (hunk 4 above)
Comment (qmonmert) [reply generators/client/templates/angular/src/main/webapp/app/layouts/profiles/page-ribbon.component.ts.ejs:22]: yes intentional

File: generators/client/templates/angular/src/main/webapp/app/login/login.component.ts.ejs (hunk 5 above)
Comment (qmonmert) [reply generators/client/templates/angular/src/main/webapp/app/login/login.component.ts.ejs:22]: yes intentional

File: generators/client/templates/angular/src/main/webapp/app/login/login.route.ts.ejs (hunk 6 above)
Comment (qmonmert) [reply generators/client/templates/angular/src/main/webapp/app/login/login.route.ts.ejs:20]: yes intentional

File: generators/client/templates/angular/src/main/webapp/app/shared/shared.module.ts.ejs (hunk 7 above)
Comment (qmonmert) [reply generators/client/templates/angular/src/main/webapp/app/shared/shared.module.ts.ejs:20]: yes intentional

**Review by kaidohallik — APPROVED** [review]
I misread changes, I interpreted added lines as deleted lines. Everything is OK. Thanks @qmonmert!

**Comment by qmonmert** [comment]
> I misread changes, I interpreted added lines as deleted lines. Everything is OK. Thanks @qmonmert!

ok :)

**Pull request merged**